### ISSUE TITLE
Missing header information for ElasticSearch 6.2

### ISSUE THREAD

**Issue by nerdstein** [knockrentals/scrapy-elasticsearch#78]
Thank you all for putting together this great tool. I was thrilled to find this.

I am currently getting an error as follows:

```
{"error":"Content-Type header [] is not supported","status":406}
```

According to this URL, https://github.com/taskrabbit/elasticsearch-dump/issues/350 , additional headers need to be passed for ElasticSearch 6.x. As follows:

```
-headers='{"Content-Type": "application/json"}
```

Could this be a new configuration added?

**Comment by nerdstein** [comment]
This is resolved by updating the elasticsearch requirement to the latest version

**Comment by jayzeng** [comment]
Yeah, I will roll in some es version detection and insert the header.
